server/index.js

```diff
@@ -28,38 +21,6 @@ function register (server, options, next) {
   // mapreduce is required for `db.query()`
   PouchDB.plugin(require('pouchdb-mapreduce'))
```

Comment:
remove this line, it should be moved to `cli/parse-options.js` like most of the rest of the code that you removed :)